services/director-v2/src/simcore_service_director_v2/models/schemas/dynamic_services/scheduler.py

```diff
@@ -223,6 +223,9 @@ def compose_spec_submitted(self) -> bool:
             "important data might be lost. awaits for manual intervention."
         ),
     )
+    wait_for_manual_intervention_logged: bool = Field(
```

Comment:
what is the difference with the variable above?

Reply:
ok now I get it. would it be possible to have sub models at some point? this class is huge... how does pylint even accept it anymore?

Reply:
At some point this class will directly go away in favour of something more manageable. I think pylint does not complain because it is a "data model". Think you are allowed to have as many attributes as you'd like on those.